Source/AliveLib/Slurg.cpp

```diff
@@ -0,0 +1,29 @@
+#include "stdafx.h"
+#include "Slurg.hpp"
+
+WORD Slurg::sSlurg_Step_Watch_Points_Idx_5C1C08;
+char Slurg::sSlurg_Step_Watch_Points_Count_5BD4DC[2];
+Slurg_Step_Watch_Points Slurg::sSlurg_Step_Watch_Points_5C1B28[2];
+
+void CC Slurg::Clear_Slurg_Step_Watch_Points_449A90()
+{
+	sSlurg_Step_Watch_Points_Idx_5C1C08 = sSlurg_Step_Watch_Points_Idx_5C1C08 == 0;
+	sSlurg_Step_Watch_Points_Count_5BD4DC[sSlurg_Step_Watch_Points_Idx_5C1C08] = 0;
+
+	Slurg_Step_Watch_Points* point = &sSlurg_Step_Watch_Points_5C1B28[sSlurg_Step_Watch_Points_Idx_5C1C08];
+	point->field_0_points[0].field_0_xPos = 0;
```

Comment:
you can use a range based for and C++ default in here, e.g

for (auto& point : sSlurg_Step_Watch_Points_5C1B28[sSlurg_Step_Watch_Points_Idx_5C1C08].field_0_points)
{
 point = {};
}